iXBRLViewerPlugin/viewer/src/less/inspector.less

```diff
@@ -708,6 +708,28 @@
       }
     }
 
+    .concept td > div {
+      display: flex;
+
+      span.text {
+        white-space: nowrap;
```

Comment:
(Arbitrary line for discussion.)

In other contexts, we have found it useful to split concept local names on heuristic word boundaries and then to insert [`<wbr>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr) between each word.

Reply:
That would certainly give neater wrapping, but in this case I think it's actually preferable to truncate, as we already have the standard label shown a little way above, and personally the only thing I ever want to do with the concept name is copy and paste it elsewhere.